tools/sidekick/config.json

```diff
@@ -96,6 +96,17 @@
       "includePaths": [ "**.docx**" ],
       "passConfig": true,
       "passReferrer": true
+    },
+    {
+      "containerId": "tools",
+      "id": "version-history",
+      "title": "Version History",
+      "environments": [ "edit" ],
+      "url": "tools/version-history?project=milo--adobecom",
```

Comment:
Looks like the `project` query param is project-specific. If that is the case, please add documentation how to configure this version history feature.

Reply:
This is actually not required. Have removed the query param.